webpack.mix.js

```diff
@@ -3,6 +3,7 @@ const mix = require('laravel-mix').mix;
 mix
   .js('src/js/main.js', 'dist/js/main.js')
   .sass('src/scss/main.scss', 'dist/css/main.css')
+  .sass("src/scss/editor.scss", "dist/css/editor.css")
```

Comment:
perhaps just for consistency use single quotes.

Reply:
Done!